src/include/msvc.h

```diff
@@ -19,8 +19,6 @@
 /*
  * Add MSVC-specific attributes and pragmas to types and function declarations.
  */
-#define	WT_COMPILER_TYPE_ALIGN(x)	__declspec(align(x))
```

Comment:
The pull request tester with the anonymous unions has run on windows and passed. Anonymous unions are supported by Windows as far back as MSVC from 2003. Is there anything specific you were looking for in testing?